src/Apps/Artist/ArtistApp.tsx

```diff
@@ -153,3 +153,6 @@ export const ArtistAppFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+// Top-level route needs to be exported for bundle splitting in the router
+export default ArtistAppFragmentContainer
```

Comment:
`@loadable/components` has a hard requirement that split bundles export a default. There's currently a [PR open](https://github.com/gregberge/loadable-components/pull/483) to fix this.

Reply:
It’s a fine tradeoff for now 👍